### ISSUE TITLE
Criar component para listar dados da api

### ISSUE THREAD

**Issue by wagnerssouza** [LeandroCarolino01/ang-project#1]

**Comment by LeandroCarolino01** [comment]
voce quer que eu crie um servico e use httpclientmodule para o get request?

**Comment by wagnerssouza** [comment]
Yes, I want a http request with verb get in any api and show in view at your component.

You can research a API in https://github.com/public-apis/public-apis

**Comment by LeandroCarolino01** [comment]
I got the images from the API but now I am trying to find out how to display more than 1 image.
